functions.js

```diff
@@ -3,10 +3,10 @@
 const mysql = require("mysql")
 
  const database = mysql.createConnection({
-  host : "localhost",
-  user : "ams_user",
-  password : "ams@123",
-  database : "ams"
+	host: process.env.DB_HOST || 'localhost',
+	user: process.env.DB_USER || 'root',
```

Comment:
why are we using root here?
I think it is better to have a custom user because in that case, all developers can use the same password, user and database. The benefit of using the same database settings is that it does not create any confusion among developers while setting up the database.

Reply:
Also, if we use the root password, then developers have to change this file every time they clone the repo, because every user has different root password and this will create merge conflicts.

Reply:
The database connection credentials have to be in a separate file The contributors create that file in their local file system and use that file to load environment variables having connections credentials. In this way, the file doesn't need to be changed every time.

Reply:
Then why you hardcoded your database settings. And I think it is better to have a standard for database settings. May be you create an environment file having the same settings as I had added in my last commit and commit that file. If we do that than it becomes easy for devlopers to setup the project. becase now they have clear instructions.

Reply:
Yes, you are right. The credentials shouldn't be hardcoded and I will remove them soon.
It is not a good practice to include database information directly in code due to some security concerns, that's why the environment file shouldn't be tracked and must be in ".gitignore".
I will soon update the Readme with the instructions to configure and run this project.

Reply:
Thanks, I will merge the PR once you removed the hardcoded code.

Reply:
I have done all the necessary changes.
1. Removed hardcoded database credentials.
2. Updates Readme.md to configure the project initially and make it run properly.